addons/arsenal/functions/fnc_handleSearchbar.sqf

```diff
@@ -17,11 +17,15 @@
 params ["_display", "_control"];
 
 private _searchString = ctrlText _control;
+if (_searchString != "") then {
+    // Sanitize to prevent script errors
```

Comment:
That's not really what is happening, it's just that the pattern is properly handling user input now.
```
    // Sanitize to prevent script errors
```

Reply:
Leftovers from local branch